docs/userGuide/contentAuthoring.md

```diff
@@ -375,6 +375,50 @@ Front matter can also be included from a separate file, just like how content ca
 
 This will result in `index.md` having the title `Binary Search Tree` and the specified keywords in order for it to be looked up through the search bar.
 
+### Inserting content into a page's head element
+
+While authoring your website, you may want to have your own CSS or Javascript file to be included in a page.
+
+- Start by creating a head file with Markdown extension (`.md`) in the `_markbind/head` folder.
+    - More than one head file can be created for different pages.
+
+- Author your `<style>` elements for CSS files and `<link>` elements for Javascript files using HTML as shown below.
+    - Ensure that any url starts from the root directory <code>{<span></span>{baseUrl}}/</code> when you are referencing your files.
+
+```css
+/* In yourCSSFolder/subfolder/myCustomStyle.css */
+p {
+    background: lightskyblue;
+}
+```
+
+```js
+// In yourScriptFolder/myCustomScript.js
+alert("Welcome to my website!"); 
+```
+
+```html
+<!-- In _markbind/head/compiledRef.md -->
+<link rel="stylesheet" href="{{baseUrl}}/yourCSSFolder/subfolder/myCustomStyle.css">
+<script src="{{baseUrl}}/yourScriptFolder/myCustomScript.js"></script>
+```
+
+- Specify the head file in pages that you want it, within the [front matter](#front-matter) `head` attribute.
+
+```html
+<!-- In the page you want the head file to be in -->
+<frontmatter>
+  head: compiledRef.md
+</frontmatter>
+```
+
+The head file contents will be placed near the end of the page's head tag.
+It will override existing Bootstrap and MarkBind CSS styles if there is an overlap. 

```

Comment:
`It will...` -> `Your head file will...`
`there is an overlap` -> `there is an overlap of classes`.

Reply:
... overlap of ~~classes~~ -> `... overlap of selectors`

[Doc here](https://www.w3schools.com/css/css_syntax.asp)